packages/devtools_app/lib/src/preferences.dart

```diff
@@ -21,15 +24,27 @@ class PreferencesController {
     }
 
     // Get the current values and listen for and write back changes.
-    final String value = await storage.getValue('ui.darkMode');
+    String value = await storage.getValue('ui.darkMode');
     _darkModeTheme.value = value == null || value == 'true';
```

Comment:
I know it's not your code, but while you're here would you mind changing this to use toggleDarkModeTheme?

Reply:
Not a problem!